gulp/gulp.d.ts

```diff
@@ -5,9 +5,11 @@
 
 /// <reference path="../node/node.d.ts" />
 /// <reference path="../orchestrator/orchestrator.d.ts" />
+/// <reference path="../vinyl/vinyl.d.ts" />
```

Comment:
gulp uses vinyl@^0.4.0, but the version of vinyl you referenced is 1.1.0. Consider changing this to
```ts
/// <reference path="../vinyl/vinyl-0.4.3.d.ts" />
```